packages/server/marketplaces/chatflows/Chat with a Podcast.json

```diff
@@ -0,0 +1,702 @@
+{
```

Comment:
IMO this template is almost same as `Conversational Retrieval QA Chain` template. Maybe a template that uses an agent + search tool?

Reply:
I have based this on [LangChainJS use case example](https://github.com/langchain-ai/langchainjs/blob/main/examples/src/use_cases/youtube/chat_with_podcast.ts).

It demonstrates the usage of a Web Search document loader (YouTube transcripts in this case). `Conversational Retrieval QA Chain` is a generalized example and this chat flow is a specific use case. 

> Maybe a template that uses an agent + search tool?

`MRKLAgent` example is already occupied. Or do you have something else in mind?

I'm happy to make changes if needed, just let me know.

Reply:
ah okay, i'm good with it 👍